tests/unit/test_workspace_work_item_types.py

```
@@ -40,7 +40,39 @@ def test_create_update_workspace_work_item_type(
         )
         assert updated.id == created.id
         assert updated.description == "Updated description"
-        # No delete endpoint on workspace WITs per spec.
+
+        try:
+            client.workspace_work_item_types.delete(workspace_slug, created.id)
+        except Exception:
+            pass
```

Ruff (0.15.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)